deis/manifests/deis-builder-rc.yaml

```diff
@@ -16,8 +16,8 @@ spec:
     spec:
       containers:
         - name: deis-builder
-          image: quay.io/deisci/builder:v2-alpha
-          imagePullPolicy: Always
+          image: quay.io/deis/builder:v2-alpha
+          imagePullPolicy: IfNotPresent
```

Comment:
I think we're viewing v2-alpha as a single version, instead of a channel. @jchauncey had suggested using `canary` as the channel name.